### PR TITLE
[IMP] CRM: improve pricing section of the doc about Outlook Extension

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: crm/optimize/outlook_extension.rst [hunk 1]
```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
```
Comment (vpk-odoo) [inline crm/optimize/outlook_extension.rst:144]: Might be a good idea to link to the articles about Lead Enrichment and/or lead generation

**Review by ksc-odoo — COMMENTED** [review]
Hope this works...

File: crm/optimize/outlook_extension.rst [hunk 2]
```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
```
Comment (sdr-odoo) [inline crm/optimize/outlook_extension.rst:143]: Where is the company insights section? Is this supposed to link somewhere?

File: crm/optimize/outlook_extension.rst (hunk 1 above)
Comment (ksc-odoo) [inline crm/optimize/outlook_extension.rst:144]: Might be good to say...

This service is part of a paid service known as **Lead Generation**.

File: crm/optimize/outlook_extension.rst [hunk 3]
```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
```
Comment (ksc-odoo) [inline crm/optimize/outlook_extension.rst:142]: Consider:

The extension is **free** and doesn't require any purchase to be installed or any entries to be created in your database.

File: crm/optimize/outlook_extension.rst [hunk 4]
```diff
@@ -137,7 +137,27 @@ Connect to your database
...
+The extension allows you to test this service for free, even if you don't connect the extension to a
+database. After a while, the extension asks you to buy credits to keep using this additional service.
+
+.. image:: media/outlook-extension-credits-message.png
+   :align: center
+   :alt: "Could not auto-complete the company: not enough credits!" warning message in the Outlook
+         extension
+
+You can disregard this message and keep using the extension for free without *Lead Enrichment* or
+buy more *Lead Generation* credits.
```
Comment (sdr-odoo) [inline crm/optimize/outlook_extension.rst:155]: I would write this sentence like this: "You can disregard this message, keep using the extension for free without *_Lead Enrichment_*, or buy more _*Lead Generation*_ credits.

File: crm/optimize/outlook_extension.rst [hunk 5]
```diff
@@ -137,7 +137,27 @@ Connect to your database
...
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
+
+The extension allows you to test this service for free, even if you don't connect the extension to a
+database. After a while, the extension asks you to buy credits to keep using this additional service.
```
Comment (vpk-odoo) [inline crm/optimize/outlook_extension.rst:147]: Might be better to say "The extension allows you to test this service for free without connecting the extension to a database. After a while, the extension asks you to buy credits to continue using this service.

File: crm/optimize/outlook_extension.rst (hunk 1 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:144]: Thanks for the idea.
There is no such doc but I added a link to the related Odoo Learn video in the `seealso` section.

File: crm/optimize/outlook_extension.rst (hunk 2 above)
Comment (AntoineVDV) [reply crm/optimize/outlook_extension.rst:143]: In RST, when text is \*wrapped in single asterisks\*, it is displayed in *italic*. When that text is \*\*wrapped in double asterixes\*\* is displayed in **bold**.

You can tell if some RST code leads somewhere (external link or internal reference) if it uses one of the formattings shown [here](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#hyperlinks).

File: crm/optimize/outlook_extension.rst (hunk 1 above)
Comment (AntoineVDV) [reply crm/optimize/outlook_extension.rst:144]: To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

File: crm/optimize/outlook_extension.rst (hunk 4 above)
Comment (AntoineVDV) [reply crm/optimize/outlook_extension.rst:155]: To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

File: crm/optimize/outlook_extension.rst (hunk 5 above)
Comment (AntoineVDV) [reply crm/optimize/outlook_extension.rst:147]: To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

File: crm/optimize/outlook_extension.rst (hunk 2 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:143]: > Where is the company insights section? Is this supposed to link somewhere?

It's a section in the extension. You can see it in one of the screenshots.

File: crm/optimize/outlook_extension.rst (hunk 1 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:144]: > Might be good to say...
> 
> This service is part of a paid service known as **Lead Generation**.

did the modification. And indeed, you can use the +- button
image

File: crm/optimize/outlook_extension.rst (hunk 4 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:155]: > I would write this sentence like this: "You can disregard this message, keep using the extension for free without __Lead Enrichment__, or buy more __Lead Generation__ credits.

Thanks, but it is not three different possibilities, but only two:
all that is before "or", and what's after.
Would a comma before "or" make it more understandable?

File: crm/optimize/outlook_extension.rst (hunk 5 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:147]: > Might be better to say "The extension allows you to test this service for free without connecting the extension to a database. After a while, the extension asks you to buy credits to continue using this service.

thanks, but it is actually two different things:
you can try it without connecting
you can try it, even when connected
and you can connect and buy

I'll see how to make this clearer. Thanks!

File: crm/optimize/outlook_extension.rst (hunk 3 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:142]: That was not the meaning of the sentence.
I simplified it:

> The extension is **free** and doesn't require any purchase to be installed and used.

File: crm/optimize/outlook_extension.rst [hunk 6]
```diff
@@ -137,7 +137,27 @@ Connect to your database
...
+database or not. After a while, the extension asks you to buy credits to keep using this additional
+service.
+
+.. image:: media/outlook-extension-credits-message.png
+   :align: center
+   :alt: "Could not auto-complete the company: not enough credits!" warning message in the Outlook
+         extension
+
+You can buy more *Lead Generation* credits to keep using this service or disregard this message and
+keep using the extension for free without *Lead Enrichment*.
```
Comment (Lau-Odoo) [inline crm/optimize/outlook_extension.rst:155]: I think it could be inside of a "tip".

**Review by Lau-Odoo — APPROVED** [review]

File: crm/optimize/outlook_extension.rst (hunk 6 above)
Comment (jcs-odoo) [reply crm/optimize/outlook_extension.rst:155]: image

**Pull request merged**